schedule/yam/agama_lvm_encrypted.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: agama_lvm_encrypted
+description: >
+  Puppeteer use encrypted lvm test on agama
```

Comment:
this is an installation, we should describe it as so.

Reply:
Give it an example to not miss it :D

Reply:
In general never mention the technology, we have switch from playwright to puppeteer and it really doesn't matter, the important thing is what it does functional-wise.
`Installation with lvm encrypted".